oci/archive/oci_src.go

```diff
@@ -38,12 +38,12 @@ func newImageSource(ctx context.Context, sys *types.SystemContext, ref ociArchiv
 }
 
 // LoadManifestDescriptor loads the manifest
-func LoadManifestDescriptor(imgRef types.ImageReference) (imgspecv1.Descriptor, error) {
+func LoadManifestDescriptor(imgRef types.ImageReference, sys *types.SystemContext) (imgspecv1.Descriptor, error) {
```

Comment:
By convention, the parameters go `context.Context, *types.SystemContext`, then others.